android/build.gradle

```diff
@@ -27,6 +27,10 @@ android {
         //noinspection OldTargetApi
         targetSdkVersion safeExtGet('targetSdkVersion', 27)
     }
+    compileOptions {
```

Comment:
I think it adds a compatibility with specific RN version - may I ask you what is the version are you using?

Reply:
I'm using 0.64.4 and if I don't set compileOptions it fails to compile. But with 0.72.3 it works fine!